packages/react/src/Hero/Hero.figma.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import {Hero} from './Hero'
+import figma from '@figma/code-connect'
+
+figma.connect(
+  Hero,
+  'https://www.figma.com/design/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=134-5526&t=hNiuAeHKuuLZeXnj-11',
+  {
+    props: {
+      heading: figma.string('heading'),
```

Comment:
FYI the text seems to have an error. 

Would this be because it should be:

```
heading: figma.string('Heading'),
```

Or should it grab that content from `nestedProps`?

```
    content: figma.nestedProps('Content', {
      headingText: figma.textContent('Heading'),
    }),

``` 

![Screenshot 2025-03-04 at 09 41 44](https://github.com/user-attachments/assets/aff4f84a-2ecb-43d2-a543-7e5be7818194)